web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx

```diff
@@ -297,7 +339,11 @@ export function SendForm(props: SendFormProps) {
           </Button>
 
           <div>
-            <Button type="submit" posIcon={<FiArrowUpRight />}>
+            <Button
+              type="submit"
+              posIcon={<FiArrowUpRight />}
+              onClick={() => confirmMnsAddress()}
```

Comment:
here the button is submit and has a onClick. It's not easy to read. Could you move the confirmMnsAddress() into the form handler?

also by the way it would have be: `onClick={confirmMnsAddress}` ;)

Reply:
great suggestion ty